Add section on sdk features to README | New Docs Bug process (#256)
grammarly/grammarly-for-developers: readme.md

[![Grammarly for Developers](https://github.com/grammarly/grammarly-for-developers/blob/main/docs/sdk_product_animation.gif)](https://developer.grammarly.com)

> ## Features of the Text Editor SDK 💻

Reviewer: ```suggestion
## About the Text Editor SDK 💻
```

> The Text Editor SDK features writing assistance to your end users across several dimensions, including:

Reviewer: ```suggestion
The Text Editor SDK provides writing assistance to your end users across several dimensions, including:
```

- **Correctness** - Eliminates grammar, spelling, and punctuation mistakes and ensures word choices sound natural and fluent.

> - **Clarity** - Helps every sentence stay concise and easy to follow and rewrites hard-to-read sentences.

Reviewer: ```suggestion
- **Clarity** - Makes every sentence concise and easy to follow and rewrites hard-to-read sentences.
```

- **Engagement** - Chooses the most compelling words and uses varied, engaging sentence structure.
- **Delivery** - Helps end users find the right tone, eliminates hedging language (to sound more confident), finesses phrases to have more tact and empathy for readers, and supports respectful and current language practices.

> **Note:** Some suggestions may only be available to paid Grammarly users who connect their account; however, writing quality features like grammar, spelling, and punctuation assistance are free for your users forever.

Reviewer: I see you pulled this from the docs, and I think we should change it there as well. I have a feeling this is left over from before we had paid plans. 

This statement is half true. Some suggestions are available for free. Others require either a paid plan or a Premium or GB user to connect their account. Here is the full statement: https://developer.grammarly.com/docs/faq#what-kinds-of-grammarly-communications-assistance-will-users-receive-in-my-app

I'm wondering if we need this statement here at all as we aren't talking about free/paid here. If we want to include something, let's clarify the statement and/or point to the faq question.

Author: Totally agree. And I flagged that in my notes on the public docs as well. 

I'm going to remove this note, for now, and re-add it when we change the copy on the developer docs site.
